modules/custom/az_metrics/az_metrics.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Quickstart Metrics
+type: module
+description: Gather and report metrics related to Arizona Quickstart.
```

Comment:
Where does the data go? Should we put a note here about that?

Reply:
I think if we have email and username we probably shouldn't publish the data.